File: 1005 - Maxunuze Sum of Array After K Negations.cpp
class Solution {
public:
    int largestSumAfterKNegations(vector<int>& nums, int k) {
        int res = 0;
        for (int i = 0; i < k; i++) {
            sort(nums.begin(), nums.end());
            nums[0] *= -1;
        }
        for (int n : nums) {
            res += n;
        }
        return res;
    }
};

File: 1005 - Maxunuze Sum of Array After K Negations.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var largestSumAfterKNegations = function (nums, k) {
    let res = 0;
    for (let i = 0; i < k; i++) {
        nums.sort((a, b) => a - b);
        nums[0] *= -1;
    }
    for (let n of nums) {
        res += n;
    }
    return res;
};